render-test/runner.cpp

```diff
@@ -392,7 +433,25 @@ bool TestRunner::runOperations(const std::string& key, TestMetadata& metadata) {
             const mbgl::JSValue* propertyValue = &operationArray[3];
             layer->setLayoutProperty(propertyName, propertyValue);
         }
-
+    // probeMemoryStart
+    } else if (operationArray[0].GetString() == memoryProbeStartOp) {
+        assert(!AllocationIndex::isActive());
+        AllocationIndex::setActive(true);
+    // probeMemory
+    } else if (operationArray[0].GetString() == memoryProbeOp) {
+        assert(AllocationIndex::isActive());
+        assert(operationArray.Size() >= 2u);
```

Comment:
should it be strictly 2?

Reply:
That's in sync with other operations parsing